src/ConfigurationHelper.php

```diff
@@ -0,0 +1,539 @@
+   * Directory entry content type.
+   *
+   * @param \Drupal\search_api\IndexInterface $index
+   *   The index to the facet field to.
+   */
+  protected function indexAddFacetField(IndexInterface $index): void {
+    if ($index->getField(Constants::FACET_INDEXING_FIELD)) {
+      return;
+    }
+
+    $field = new SearchIndexField($index, Constants::FACET_INDEXING_FIELD);
+    $field->setLabel('Facets');
+    $field->setDataSourceId('entity:node');
+    $field->setPropertyPath(Constants::FACET_SELECTION_FIELD);
+    $field->setType('integer');
+    $field->setDependencies([
+      'config' => [
+        'field.storage.node.' . Constants::FACET_SELECTION_FIELD,
+      ],
+    ]);
+    $index->addField($field);
+  }
+
+  /**
+   * Setup indexing on the Title Sort field of Directory entries.
+   *
+   * @param \Drupal\search_api\IndexInterface $index
+   *   The index to the title sort field field to.
+   */
+  protected function indexAddTitleSortField(IndexInterface $index): void {
+    if ($index->getField(Constants::TITLE_SORT_FIELD)) {
+      return;
+    }
+
+    $field = new SearchIndexField($index, Constants::TITLE_SORT_FIELD);
+    $field->setLabel('Title (sort)');
+    $field->setDataSourceId('entity:node');
+    $field->setPropertyPath(Constants::TITLE_SORT_FIELD);
+    $field->setType('string');
+    $field->setDependencies([
+      'config' => [
+        'field.storage.node.' . Constants::TITLE_SORT_FIELD,
+      ],
+    ]);
+    $index->addField($field);
+  }
+
+  /**
+   * Setup indexing on the Directory channels field of Directory entries.
+   *
+   * @param \Drupal\search_api\IndexInterface $index
+   *   The index to the channel field to.
+   */
+  protected function indexAddChannelsField(IndexInterface $index): void {
+    if ($index->getField(Constants::CHANNEL_SELECTION_FIELD)) {
+      return;
+    }
+
+    $field = new SearchIndexField($index, Constants::CHANNEL_SELECTION_FIELD);
+    $field->setLabel('Directory channels');
+    $field->setDataSourceId('entity:node');
+    $field->setPropertyPath(Constants::CHANNEL_SELECTION_FIELD);
+    $field->setType('string');
+    $field->setDependencies([
+      'config' => [
+        'field.storage.node.' . Constants::CHANNEL_SELECTION_FIELD,
+      ],
+    ]);
+    $index->addField($field);
+  }
+
+  /**
+   * Import config entity for the directory Facet.
+   */
+  public function createFacet(): void {
+    if ($this->entityTypeManager->getStorage('facets_facet')->load(Constants::FACET_CONFIG_ENTITY_ID)) {
+      return;
+    }
+
+    $conditional_config_path = $this->moduleExtensionList->getPath('localgov_directories') . '/config/conditional';
+    if ($this->importConfigEntity('facets_facet', $conditional_config_path, Constants::FACET_CONFIG_FILE)) {
+      $this->configInstaller->installOptionalConfig();
+    }
+  }
+
+  /**
+   * Update a block's visibility to add to content type.
+   *
+   * The given block should appear sidebar pages for the given content type.
```

Comment:
s/sidebar pages/in the sidebars of pages/

Reply:
Corrected in https://github.com/localgovdrupal/localgov_directories/pull/233/commits/b187781ad0067d98be3ee637f3ef155c6c967ba2